extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerBuildTimeConfig.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.narayana.jta.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+public final class TransactionManagerBuildTimeConfig {
+    /**
+     * Allow using multiple resources in the same transactional demarcation with XA.
+     * <p>
+     * This is UNSAFE and may only be used for compatibility.
+     * Either use XA if you want consistency or split the code in two separate methods.
+     * <p>
+     * This property will be removed in 3.11 so you should plan fixing your application accordingly.
+     *
+     * @deprecated This property is going to be removed in Quarkus 3.11.
+     */
```

Comment:
I'd suggest more explanatory text here, something similar to the following:

```
     * Allow using multiple XA unaware resources in the same transactional demarcation with XA capable resources.
     * This is UNSAFE and may only be used for compatibility.
     * <p>
     * Either use XA for all resources if you want consistency, or split the code into separate
     * methods with separate transactions.
     * 
     * Note that using a single XA unaware resource together with XA aware resources, known as
     * the Last Resource Commit Optimization (LRCO), is different from using multiple XA unaware
     * resources. Although LRCO allows most transactions to complete normally, some errors can
     * cause an inconsistent transaction outcome. Using multiple XA unaware resources is not
     * recommended since the probability of inconsistent outcomes is significantly higher and
     * much harder to recover from than LRCO. For this reason, use LRCO as a last resort.
     * <p>
     * This property will be removed in 3.11 so you should plan fixing your application accordingly.
```